packages/neuron-ui/src/components/BalanceSyncingIcon/balanceSyncIcon.module.scss

```diff
@@ -1,7 +1,17 @@
 .confirm {
   margin-left: 4px;
+  animation: spin 2s linear infinite;
+
+  @keyframes spin {
```

Comment:
May be we can move `spin` to common scss?

Reply:
updated.